backend/tests/services/test_summary_service.py

```
@@ -0,0 +1,484 @@
+                    success=True,
+                    parsed_content={
+                        "short_summary": "New summary",
+                        "long_summary": "New longer summary",
+                        "topics": ["new_topic"],
+                    },
+                    task_id=123,
+                )
+            )
+            mock_executor.return_value = mock_instance
+
+            result = await summary_service.trigger_kb_summary(
```

Ruff (0.14.11)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)